docs/conf.py

```diff
@@ -63,7 +63,7 @@ def setup(sphinx):
 
 # General information about the project.
 project = 'Solidity'
-project_copyright = '2016-2021, Ethereum'
+project_copyright = '2016-2023, The solidity Authors'
```

Comment:
Maybe we could just drop the end date? AFAIK it's not required.
```suggestion
project_copyright = '2016, The solidity Authors'
```

Reply:
I personally prefer having end dates: with such end date it is more clear until when a project was actively maintained.

Reply:
Unless we forget to update it anyway :)

If we decide to keep it, maybe we should add a point to release checklist to take a look at it from time to time.

Reply:
In my previously workplace, this was done by an automated script at the beginning of the year, i.e .the bot would open a PR, and someone would go over it and merge if it's all good. Although we were required to have the copyright disclaimed with the appropriate dates in each file, so this may be a bit of an overkill for us.